paper-autocomplete.html

```diff
@@ -128,7 +128,7 @@
     <div class="input-wrapper" role="combobox" aria-haspopup="true" aria-owns="suggestionsWrapper" aria-expanded$="[[_isSuggestionsOpened]]">
       <!-- For accessibility, it is needed to have a label or aria-label. Label is preferred -->
       <label for="autocompleteInput" class="sr-only">[[label]]</label>
-
+      <paper-input   name$="[[name]]" value="{{_selectedOption.value}}" hidden ></paper-input >
```

Comment:
This hidden input will represent the given name on paper-autocomplete, It will get the selected value of the component. On form submit, this will make the selected value present in form data.
It just my work around, never mind.